services/api-server/src/simcore_service_api_server/models/schemas/functions.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=protected-access
+
+from enum import StrEnum
+from typing import Annotated, Final
+
+from models_library.functions import FunctionID, FunctionJobCollectionID, FunctionJobID
+from pydantic import BaseModel, ConfigDict, Field
+from servicelib.celery.models import TaskState
+
+_JOB_TASK_RUN_STATUS_PREFIX: Final[str] = "JOB_TASK_RUN_STATUS_"
+
+
+class FunctionJobsListFilters(BaseModel):
+    """Filters for listing function jobs"""
+
+    function_id: Annotated[
+        FunctionID | None,
+        Field(
+            description="Filter by function ID pattern",
+        ),
+    ] = None
+
+    function_job_ids: Annotated[
+        list[FunctionJobID] | None,
+        Field(
+            description="Filter by function job IDs",
+        ),
+    ] = None
+
+    function_job_collection_id: Annotated[
+        FunctionJobCollectionID | None,
+        Field(
+            description="Filter by function job collection ID",
+        ),
+    ] = None
+
+    model_config = ConfigDict(
+        extra="ignore",
+    )
+
+
+class FunctionJobCreationTaskStatus(StrEnum):
+    PENDING = f"{_JOB_TASK_RUN_STATUS_PREFIX}PENDING"
+    STARTED = f"{_JOB_TASK_RUN_STATUS_PREFIX}STARTED"
+    RETRY = f"{_JOB_TASK_RUN_STATUS_PREFIX}RETRY"
+    SUCCESS = f"{_JOB_TASK_RUN_STATUS_PREFIX}SUCCESS"
+    FAILURE = f"{_JOB_TASK_RUN_STATUS_PREFIX}FAILURE"
```

Comment:
The constant is longer that the value funny.
Q: could it be a good idea to have the Prefix initial part different from the one used for NOT_YET_SCHEDULED and ERROR?